distributed/tests/test_reschedule.py

```diff
@@ -81,15 +80,15 @@ def f(x):
     assert all(f.key in b.data for f in futures)
 
 
-@pytest.mark.parametrize("long_running", [False, True])
+@pytest.mark.parametrize("state", ["executing", "long-running"])
 @gen_cluster(client=True, nthreads=[("", 1)])
-async def test_cancelled_reschedule(c, s, a, long_running):
+async def test_cancelled_reschedule(c, s, a, state):
     """A task raises Reschedule(), but the future was released by the client"""
```

Comment:
nit: I'd prefer a bi-directional reference so that someone touching `test_cancelled_reschedule` will also know to take a look at adjusting `test_cancelled_reschedule_worker_state`.
```suggestion
    """A task raises Reschedule(), but the future was released by the client.
    Same as test_cancelled_reschedule_worker_state"""
```